packages/date/package.json

```diff
@@ -23,7 +23,7 @@
     "test:ci": "tsdx test --env=jsdom --ci"
   },
   "dependencies": {
-    "@contentful/f36-components": "^4.0.27",
```

Comment:
r u sure 🙄 ?

Reply:
Yes, this is the newest version https://github.com/contentful/forma-36/releases?page=1